core/collection-factory/mapdb3/src/main/java/org/eclipse/rdf4j/collection/factory/mapdb/MapDb3CollectionFactory.java

```diff
@@ -36,6 +40,7 @@
 import org.eclipse.rdf4j.model.Value;
 import org.eclipse.rdf4j.query.BindingSet;
 import org.eclipse.rdf4j.query.MutableBindingSet;
+import org.jetbrains.annotations.NotNull;
```

Comment:
This seems like a accident? we have not imported this one yet.

Reply:
Yeah, thought I had clean that up.